warehouse/query-core/src/main/java/datawave/query/scheduler/PushdownFunction.java

```diff
@@ -167,58 +144,24 @@ protected void redistributeQueries(Multimap<String,QueryPlan> serverPlan, Tablet
         
     }
     
-    protected Map<String,Map<KeyExtent,List<Range>>> binRanges(TabletLocator tl, Instance instance, List<Range> ranges) throws AccumuloException,
-                    AccumuloSecurityException, TableNotFoundException {
-        Map<String,Map<KeyExtent,List<Range>>> binnedRanges = new HashMap<>();
-        
-        int lastFailureSize = Integer.MAX_VALUE;
-        
-        while (true) {
```

Comment:
I think we potentially lost some functionality here.  The retry until we have a stable state of tablet locations was critical because this is working outside of a tserver, and hence rfiles are continually moving.  We wanted some sort of certainty that we have a stable state before launching the scans.  If the tableOperations().locate(...) does the same thing underneath then I am ok with this.  Otherwise I would prefer seeing if we could do this in a similar manner with the 2.0 API (e.g. persistent client connection).

Reply:
I agree.  I will port over the retry functionality and look further into the behavior of the tableOps.locate() method. 

I still think this class is a good opportunity to migrate critical code to Accumulo API, ahead of a full upgrade to 2.0.

Reply:
Sorry I forgot I had already looked into this...

Just from looking at the implementation of ```locate()``` I believe it behaves the same.  Meaning, it will keep trying to find locations until there are no more failures.  Take a look at [this loop](https://github.com/apache/accumulo/blob/2ca12d9b2bcfe3c199ce355648bf7cd2a6ccf55d/core/src/main/java/org/apache/accumulo/core/clientImpl/TableOperationsImpl.java#L1753).